src/data/fonts.ts

```diff
@@ -0,0 +1,8 @@
+// In TypeScript 2.4 this can turn into an Enum
+// https://github.com/Microsoft/TypeScript/pull/15486
+
+export default {
+  "garamond-regular": "AGaramondPro-Regular",
+  "garamond-italic": "AGaramondPro-Italic",
+  "avant-garde-regular": "Avant Garde Gothic ITCW01Dm",
+}
```

Comment:
ATM all our font references are floating strings, this at least ties it together till TS 2.4.0